app/controllers/devise_token_auth/omniauth_callbacks_controller.rb

```diff
@@ -156,19 +156,19 @@ def set_random_password
 
     def create_auth_params
       @auth_params = {
-        auth_token:     @token,
-        client_id: @client_id,
-        uid:       @resource.uid,
-        expiry:    @expiry,
-        config:    @config
+        auth_token: @token.token,
```

Comment:
I'm thinking that you can create a method in the token factory `auth_params`, but I don't if would be so useful